tests/models/chameleon/test_modeling_chameleon.py

```diff
@@ -116,7 +116,7 @@ def prepare_config_and_inputs(self):
 
         input_mask = None
         if self.use_input_mask:
-            input_mask = torch.tril(torch.ones(self.batch_size, self.seq_length)).to(torch_device)
+            input_mask = torch.tril(torch.ones_like(input_ids).to(torch_device))
```

Comment:
common pattern: `input_mask`, which was then passed around as `attention_mask`, was a `torch.float32` and not a `torch.long` 👀